TechnicoBackend/Repositories/PropertyRepository.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using TechnicoBackend.Data;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Repositories
+{
+    public class PropertyRepository
```

Comment:
Consider using an interface IPropertyRepository from which the current class would be inherited.